camel/societies/workforce/workforce.py

```
@@ -598,3 +599,227 @@ def clone(self, with_memory: bool = False) -> 'Workforce':
+                    "result": result_task.result or "",
+                    "subtasks": [
+                        {
+                            "id": subtask.id,
+                            "content": subtask.content,
+                            "state": str(subtask.state),
+                            "result": subtask.result or "",
+                        }
+                        for subtask in (result_task.subtasks or [])
+                    ],
+                }
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)